lib/spack/spack/test/cmd/url.py

```diff
@@ -123,6 +124,10 @@ def test_url_summary(mock_packages):
     assert out_correct_versions == correct_versions
 
 
+@pytest.mark.skipif(
+    sys.platform.startswith("win"),
+    reason="Unsupported on Windows for now"
```

Comment:
As a general rule, wherever possible it would be helpful to be as descriptive as possible in the error message (e.g. indicating how difficult the problem will be to fix). Even something like

`reason="Fails on Windows (summary of error)"` would be useful

Reply:
Addressed as per our discussion.